card/db/migrate/20150220141031_create_deck_routes_file.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+
+class CreateDeckRoutesFile < ActiveRecord::Migration
+  def up
+    routes_path = File.join Wagn.root, 'config', 'routes.rb'
+    if !File.exist? routes_path
+      template_path = File.join Wagn.gem_root, 'lib', 'wagn', 'generators', 'wagn', 'templates', 'config', 'routes.erb'
+      @include_jasmine_engine = false
+      decko_path = ''
+      routes = ERB.new(File.read template_path).result binding
+      File.open(routes_path, "w") { |f|  f.write routes }
+    end
+  end
+end
```

Comment:
Yes!

If you generate a standard rails engine, you'll get:
````
module Blorgh
  class Engine < ::Rails::Engine
  end
end
````

if you generate a decko engine, you'll get:
````
module Blorgh
  class Engine < ::Decko::Engine
     # here' how you add recaptcha to decko:
     #config.recaptcha_keys blah blah blah
  end
end
````
I wouldn't go so far as to suggest this is the only possible way, but it's highly desirable.  That's how you configure the engine; it's how you give people a good running start on an engine deck by giving them editable defaults.  And if you decide, "hey, I actually think I want this to be a stand-alone wagn now (an app deck), then you can just change "Engine" to "Application", move a few files around, and roll on.

If there are major distinctions between what app decks and engine decks want the defaults of Decko::Engine to be, then we can set them in decko (gem) and override them in decko-rails.